netcloud/README.md

```diff
@@ -0,0 +1,151 @@
+# NetCloud
+
+- [Recon](#recon)
+- [Decrypt NetCloud Traffic in Wireshark](#decrypt-netcloud-traffic-in-wireshark)
+- [MITM NetCloud Traffic with mitmproxy](#mitm-netcloud-traffic-with-mitmproxy)
+- [Disconnect any Cradlepoint Router from NetCloud](#disconnect-any-cradlepoint-router-from-netcloud)
+- [RCE](#rce)
+
+## Recon
+
+Cradlepoint documentation:
+
+- [How to connect a router to NetCloud](https://customer.cradlepoint.com/s/article/NCM-Connecting-Devices)
+- [NetCloud API to query routers remotely](https://customer.cradlepoint.com/s/article/NCM-APIv2-Overview)
+  - [API query examples](https://developer.cradlepoint.com/)
+- [Enable remote administration](https://customer.cradlepoint.com/s/article/NCOS-How-to-Configure-Remote-Administration-on-a-Cradlepoint-Router)
+
+WPC Authentication / ECM:
+
+- https://accounts.cradlepointecm.com/
+  - Login with username and password
+  - Or login with `token_id` and `token_secret`
+    - Token stored in NOR dump
+    - `$ binwalk –extract nordump_pw0000.bin`
+    - `$ grep -rl "wpc.auth" .`
+- `ecm` command in the router's CLI
+  - Privileged sub-commands: `tri`, `ale`, `dif`
+- Connection
+  - Server: `stream.cradlepointecm.com`
+  - Port: `8001`
+  - Certificate: `/service_manager/services/wpcclient/stream.crt`
+- "Insecure Activation" (see `wpcclient.pyc`) probably only works if it's manually activated from the NetCloud side (see [use case]( https://customer.cradlepoint.com/s/article/How-To-Have-Cradlepoint-register-a-router-to-NetCloud-Manager-when-it-is-physically-not-accessible))
```

Comment:
Insecure activation was also used to generate temporary credentials for brand-new devices (fixed by Cradlepoint after our report, fix tested by me).

Reply:
Added.